README.md

```diff
@@ -1,4 +1,44 @@
 # Envoy Proxy Rate Limiter Cucumber
-Cucumber integration tests for the Envoy Proxy Rate Limiter 
+Cucumber integration tests for the Envoy Proxy Rate Limiter
+
+This project comprises 3 features
+
+1.  Limiter Test (limiter-test.feature)
+    This feature runs through all the requirements listed in CR-
+    It runs through with no cross-over with any other tests in this Feature
+    
+1.  Roll-Over Test (roll-forward.feature)
+    This feature re-runs the above tests except with the clock run into the next hour segment.
+    It achieves this by either waiting on the real limiter, or if it's > 15 minutes to the hour it throws Pending Exceptions
+    Or if running against the MockClient (export USE_MOCK_CLIENT=TRUE) locally, it rolls the MockClient forward an hour, so these tests can be run.
+
+3. Cross-Over Test (limiter-test-combinations.feature)
+    This test runs all of the requirements, but maintains values to the next test(s) in order to test multiple limits together.
+    
+Mock Client and Mock Limiter
+
+    Mock CLient
```

Comment:
Putting the Mock Client and Mock Limiter description in a coding block makes them scroll off the page and harder to read. Prefer if it is done in normal paragraphs / headings.

Reply:
It's the tabbing that's done that - weird - nothing to do with me